src/data_loading/test_data.py

```diff
@@ -0,0 +1,115 @@
+"""
+This module contains functions that supply simple test data.
+"""
+
+from typing import Tuple
+
+import geopandas as gpd
+import shapely
+
+from src.constants import CHERNOBYL_COORDS_UTM35N, UTM35N
+
+
+def get_square_row(
+    num_squares: int = 2,
+    translation: Tuple[float, float] = (2, 0),
+    square_side_len: float = 1,
+    origin: Tuple[float, float] = (0, 0),
+    crs: str = UTM35N,
+) -> gpd.GeoDataFrame:
+    """
+    Creates a row of squares with side length `square_side_len`,
```

Comment:
Nice and clear docstrings 👍  - see comment on PEP-257 in `chernobyl.py` for further best practices.